### ISSUE TITLE
MouseTracker.destroy() causes error

### ISSUE THREAD

**Issue by ErikDahlinghaus** [openseadragon/openseadragon#1600]
OpenSeadragon Version: `2.4.0`

When calling `.destroy()` on `MouseTracker` I get

```
Uncaught TypeError: Cannot read property 'tracking' of undefined
    at stopTracking (openseadragon.js:4458)
    at $.MouseTracker.destroy (openseadragon.js:3371)
```

Here is a snippet of setup

```javascript
this.mouseTracker = new OpenSeadragon.MouseTracker({
    element: viewerNode,
    moveHandler: (event) => {
        console.log(event.position);
    }
});

// Some time later
this.mouseTracker.destroy();
```

**Comment by msalsbery** [comment]
Thanks for reporting!

**Comment by msalsbery** [comment]
Fixed for v2.5.0 (#1872)

Feel free to reopen if issue persists!
